cobalt/dom/captions/system_caption_settings.cc

```diff
@@ -473,29 +477,40 @@ bool SystemCaptionSettings::is_enabled() {
 
 void SystemCaptionSettings::set_is_enabled(bool active) {
   DCHECK(supports_set_enabled());
+#if SB_API_VERSION >= 16
+  auto accessibility_api = static_cast<const CobaltExtensionAccessibilityApi*>(
```

Comment:
Nitpick: Should rename to StarboardExtension here as well

Reply:
Renamed